### PR TITLE
Add device-scanner-daemon

### PR DESCRIPTION
**Pull request by jgrund**
- Update block-device-listener to emit a serialized DU.
  - Add scripts for rollups.

Signed-off-by: Joe Grund <joe.grund@intel.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+language: csharp
```
Comment (johnsonw) [inline .travis.yml]: I guess they don't support F#?

File: packages/block-device-listener/lib/listener.fs [hunk 2]
```diff
@@ -1,17 +1,27 @@
+// Copyright (c) 2017 Intel Corporation. All rights reserved.
```
Comment (johnsonw) [inline packages/block-device-listener/lib/listener.fs]: So this is the new copyright?

File: packages/device-scanner-daemon/package.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  "version": "0.0.0",
+  "description": "A persistent process that consumes udev events over a unix domain socket.",
+  "main": "dist/device-scanner-daemon",
+  "scripts": {
+    "prepublish": "old_pwd=$PWD; cd ../../; dotnet fable npm-run rollup --args \"-i $old_pwd/lib/server.fs -o $old_pwd/$npm_package_main\" --port 'free'; chmod +x $old_pwd/$npm_package_main"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": "https://github.com/intel-hpdd/device-scanner/tree/master/packages/device-scanner-daemon",
```
Comment (johnsonw) [inline packages/device-scanner-daemon/package.json]: What is the "tree/master" for?

File: .travis.yml (hunk 1 above)
Comment (jgrund) [inline .travis.yml]: They do, this is how you test F#

File: packages/block-device-listener/lib/listener.fs (hunk 2 above)
Comment (jgrund) [inline packages/block-device-listener/lib/listener.fs]: Yes.

File: packages/device-scanner-daemon/package.json (hunk 3 above)
Comment (jgrund) [inline packages/device-scanner-daemon/package.json]: It's how we can link to sub-repos.

**Pull request merged**